src/index.js

```diff
@@ -8,6 +8,7 @@ import checkSignatureAndParse from './checkSignatureAndParse';
 import lineClient from './lineClient';
 import handleInput from './handleInput';
 import botimize from 'botimize';
+import { uploadImageFile, uploadVideoFile} from './fileUpload';
```

Comment:
Hmm sea that I forgot to run lint as part of CI build.

No worries, I'll cover that after merge.

Reply:
Thanks!